modules/user-guide/pages/installing-vale-cli.adoc

```diff
@@ -11,33 +11,41 @@ Install the Vale linter and the `RedHat` package to automate reviewing content f
 
 [IMPORTANT]
 ====
-Vale v3.0.0+ introduces a breaking change in how the Styles path is configured.
-Until we reconcile these changes, use Vale v2.30.0 or lower.
+Vale v3 introduced a breaking change for the Vale vocabulary folder location.
+For Vale v3+, if you use a vocabulary with your project, ensure that the vocabulary folder matches this folder structure:
+
+[source,terminal]
+----
+.vale/styles/config/vocabularies/<project_vocab_folder>
+----
 ====
 
 .Prerequisites
 
 * Install the link:https://docs.asciidoctor.org/asciidoctor/latest/install/[Asciidoctor] Ruby gem.
 
-* Remove any previously installed `mczernek/vale` RPM installation. Fedora 39 CSB is not compatible with the latest `mczernek/vale` RPM.
-To remove a previous Vale RPM installation, do the following:
+.Procedure
+
+. Install Vale.
+For RHEL or Fedora Linux operating systems, run the following command:
 +
 [source,terminal]
 ----
-sudo dnf remove vale
+sudo dnf copr enable mczernek/vale && sudo dnf install vale
 ----
 
-.Procedure
-
-. To install the Vale v2.30.0 command-line tool on RHEL or Fedora, run the following commands:
+. Verify the Vale install location.
+You might need to update your text editor with this value.
```

Comment:
A bit vague, perhaps include something like the following? 

Ensure the Vale configuration in your text editor points to the directory for the Vale executable. Run the following command to check the directory for the Vale executable:
$ which vale